hack/test-end-to-end.sh

```diff
@@ -261,8 +258,13 @@ osc process -n test -f examples/sample-app/application-template-stibuild.json >
 osc process -n docker -f examples/sample-app/application-template-dockerbuild.json > "${DOCKER_CONFIG_FILE}"
 osc process -n custom -f examples/sample-app/application-template-custombuild.json > "${CUSTOM_CONFIG_FILE}"
 
+# Client setup (log in as e2e-user and set 'test' as the default project)
+echo "[INFO] Logging in as a regular user (e2e-user:pass) with project 'test'..."
+osc login -u e2e-user -p pass
```

Comment:
Or let someone pass --context to specifically prevent the credentials from being rewritten.